### ISSUE TITLE
sqlite DB locked when you specify flywayDriver

### ISSUE THREAD

**Issue by reactormonk** [flyway/flyway#1318]
###### What version of Flyway are you using?

4.0.1
###### What database are you using (type & version)?

sqlite / xerial / 3.8.11.2
###### What operating system are you using?

Linux
###### What did you do?

```
libraryDependencies ++= Seq(
  "org.xerial" % "sqlite-jdbc" % "3.8.11.2"
)

flywayUrl := "jdbc:sqlite:buddy.db"
flywayDriver := "org.sqlite.JDBC"
```

With a random migration, e.g.

```
create table user
(
  id integer primary key,
);
```

Then run

```
sbt flywayMigrate
```
###### What did you expect to see?

Migration.
###### What did you see instead?

```
Caused by: java.sql.SQLException: database is locked
```

That only happens when I specify

```
flywayDriver := "org.sqlite.JDBC"
```

without it, it works just fine.

**Comment by kinow** [comment]
Same was happening for me, saved by @axelfontaine 's comment about not setting the JDBC driver. That indeed fixed the issue for me.

Here's a simple example:
https://github.com/kinow/matchan/tree/277d1163016594279bf575d69ac62a95e94c2b8e/databases/reminders

Checking out that directory, and running flyway migrate will create a database.sqlite in the current directory. If you edit flyway.conf, uncommenting the driver, you'll get the error.

Thanks @axelfontaine !

**Comment by kinow** [comment]
Checked out the latest code, used instructions in this [SO question]() to build the project, then in Eclipse copied the original installation drivers and jars directories over to flyway-commandline maven module.

Was ready to debug the issue, but actually, when I run the migrate command with the same scenario, it works.

So I believe it has been fixed in the trunk already :-)

My checkout revision: 8b9dbfa1dc603338d7fba64c0bda973f89003952

Hope that helps.

**Comment by axelfontaine** [comment]
This has now been fixed as part of #1499 (see my comment on that issue for the cause)

Thanks for investigating!
